spec/services/tweet_stat_adjustment_service_spec.rb

```diff
@@ -0,0 +1,78 @@
+require "rails_helper"
+
+RSpec.describe TweetStatAdjustmentService do
+  let(:travel_time) { TweetStatAdjustmentService::ADVANCE_IN_HOURS }
+
+  def build_article(*args)
+    create(:article, *args)
+  end
+
+  def travel(distance, &block)
```

Comment:
This is not really needed.

Timecop.travel accepts a block, the same for freeze